es_ES/LC_MESSAGES/archive.po

```diff
@@ -5,235 +5,235 @@ msgstr ""
 
 msgid "HISTORY"
-msgstr "History"
+msgstr "Historia"
 
 msgid "HOMEPAGE_CREATOR_STAT"
-msgstr "1.2 million"
+msgstr "1.2 millones"
 
 msgid "HOMEPAGE_DATA_HEADER"
-msgstr "Total Data"
+msgstr "Datos en total"
 
 msgid "HOMEPAGE_FLIPNOTE_STAT"
-msgstr "44 million"
+msgstr "44 millones"
 
 msgid "HOMEPAGE_FROG_DIALOG"
-msgstr "Keep on Flipnoting!"
+msgstr "¡Seguid Flipnoteando!"
 
 # again, wrapped in links
 msgid "HOMEPAGE_INTRO_FORMAT"
-msgstr "A comprehensive archive of %sFlipnote Studio%s animations posted to %sFlipnote Hatena%s before its closure in May 2013."
+msgstr "Un archivo completo de animaciones de %sFlipnote Studio%s subidas al sitio %sFlipnote Hatena%s antes de su cierre en mayo 2013."
 
 msgid "HTTP_ERROR_404_TEXT"
-msgstr "The page you were looking for could not be found."
+msgstr "No se pudo encontrar la página que buscabas. :("
 
 # Error 404
 msgid "HTTP_ERROR_FORMAT"
 msgstr "Error %s"
 
 # May 17th, 1996
 msgid "MONTH_DAY_YEAR_FORMAT"
-msgstr "F jS, Y"
+msgstr "jS F, Y"
 
 # May 1996
 msgid "MONTH_YEAR_FORMAT"
 msgstr "F Y"
 
 msgid "NAVBAR_DONATE"
-msgstr "Donate"
+msgstr "Donar"
 
 msgid "NAVBAR_FAQS"
 msgstr "FAQs"
 
 msgid "NAVBAR_HISTORY"
-msgstr "History"
+msgstr "Historia"
 
 msgid "NAVBAR_HOME"
-msgstr "Home"
+msgstr "Inicio"
 
 msgid "NAVBAR_TAGS"
-msgstr "Tags"
+msgstr "Etiquetas"
 
 msgid "NEXT"
-msgstr "Next"
+msgstr "Siguiente"
 
 msgid "NUMBER_OF_SONGS_FORMAT"
-msgstr "%d songs"
+msgstr "%d canciones"
 
 msgid "ORIGINAL_FLIPNOTE"
-msgstr "Original Flipnote"
+msgstr "Flipnote original"
 
 msgid "ORIGINAL_NOT_FOUND"
-msgstr "The original couldn't be found"
+msgstr "No se ha podido encontrar el original"
 
 msgid "PARENT_FLIPNOTE"
-msgstr "Parent Flipnote"
+msgstr "Flipnote de origen"
 
 msgid "POPULAR"
 msgstr "Popular"
 
 msgid "POPULAR_FLIPNOTES"
-msgstr "Popular Flipnotes"
+msgstr "Flipnotes populares"
 
 msgid "POPULAR_FLIPNOTES_DESC"
-msgstr "Flipnotes that have received a lot of attention from archive visitors"
+msgstr "Flipnotes que han recibido mucha atención por los visitantes del archivo"
 
 msgid "PRESS"
-msgstr "Press"
+msgstr "Prensa"
 
 msgid "PRESS_INQUIRIES"
-msgstr "Press Inquiries"
+msgstr "Consultas de prensa"
 
 msgid "PREV"
 msgstr "Prev"
 
 msgid "PURPLE_STARS"
-msgstr "Purple Stars"
+msgstr "Estrellas moradas"
 
 msgid "RANDOM"
-msgstr "Random"
+msgstr "Aleatorio"
 
 msgid "RANDOM_FLIPNOTES"
-msgstr "Random Flipnotes"
+msgstr "Flipnotes aleatorias"
 
 msgid "RANDOM_FLIPNOTES_DESC"
-msgstr "A series of Flipnotes chosen from a random point in time"
+msgstr "Un conjunto de Flipnotes elegidos de un punto aleatorio en el tiempo"
 
 msgid "RED_STARS"
-msgstr "Red Stars"
+msgstr "Estrellas Rojas"
 
 msgid "REGION_AMERICAS"
-msgstr "Americas"
+msgstr "América"
 
 msgid "REGION_EUROPE_AND_OCEANIA"
-msgstr "Europe & Oceania"
+msgstr "Europa y Oceanía"
 
 msgid "REGION_JAPAN"
-msgstr "Japan"
+msgstr "Japón"
 
 msgid "ROOT_FLIPNOTE"
-msgstr "Root Flipnote"
+msgstr "Flipnote principal"
```

Comment:
This should be changed to "Flipnote original" in order to be consistent with ORIGINAL_FLIPNOTE, or ORIGINAL_FLIPNOTE should be changed to match this. I'm not sure why there are multiple strings but it's what we have.